packages/lumx-react/src/components/fieldset/Fieldset.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from 'react';
+
+import classNames from 'classnames';
+
+import { COMPONENT_PREFIX } from '@lumx/react/constants';
+import { IGenericProps, getRootClassName } from '@lumx/react/utils';
+
+/**
+ * Defines the props of the component.
+ */
+interface IFieldsetProps extends IGenericProps {
+    legend?: ReactElement | string;
+}
+
+const COMPONENT_NAME = `${COMPONENT_PREFIX}Fieldset`;
+
+/**
+ * The default class name and classes prefix for this component.
+ *
+ */
+const CLASSNAME = getRootClassName(COMPONENT_NAME);
+
+const Fieldset: React.FC<IFieldsetProps> = ({
+    className = '',
+    children,
+    legend,
+    ...props
+}: IFieldsetProps): React.ReactElement => (
+    <fieldset className={classNames(className, CLASSNAME)} {...props}>
```

Comment:
I think you need to manage this by using the `handleBasicClasses` util from the `@lumx/react/utils` package. you can take a look at [this example](https://github.com/lumapps/design-system/blob/master/packages/lumx-react/src/components/autocomplete/Autocomplete.tsx#L244) :)